server/scim/group_template.py

```diff
@@ -36,7 +36,26 @@ def link(name: str, value: str):
         scim_sram_extension['labels'] = sorted(labels)
 
     links = []
-    for name in ['logo', 'website_url']:
+
+    # This group object is either a Group or a Collaboration (since this is a Union)
+    # Now I need to make a link reference only in case this object is a Collaboration
+    # Maybe that can be determined anyway smarter than this...
+    collaboration = None
+    try:
+        # Will not throw exception if this object is actually a group with a
+        # foreign key reference to the collaboration...
+        _ = group.collaboration
+    except Exception:
+        # Ok, Exception was thrown, this object was actually a collaboration !
+        collaboration = group
+
+    if collaboration:
+        links.append(
+            link('sbs_url', f"{application_base_url()}/collaborations/{collaboration.identifier}")
+        )
+
+    # Add additional links, at this moment only 'logo'
+    for name in ['logo', ]:
```

Comment:
Waarom worden `logo` and `sbs_url` hier nu anders behandeld?  Beide moeten alleen worden gezet als het een CO is, toch?  Een groep heeft niet eens een logo...

Reply:
Je hebt helemaal gelijk. Group heeft geen Logo. Ik had op zich de code voorbereid op mogelijke 'links' naar items die zowel in Groups als in Collaborations kunnen voorkomen. De safety zit in gebruik van getattr.
Maar zoals de stand van zaken nu is, hebben we inderdaad geen items uit het Group ocject waarvan we een link willen persisteren in de SCIM Group extension links. Zeg maar, liever code aanpassen of zo laten opp mogelijk toekomstig gebruik ?

Reply:
Maak het maar gewoon zo expliciet mogelijk voor de huidige situatie.  Ik voorzie ook niet extra informatie over groepen